pages/Page.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export default class Page extends React.Component {
+  static async getInitialProps({ query, asPath }) {
+    const route = asPath.split('/').length > 1 ? asPath.split('/')[1] : '/';
```

Comment:
I wouldn't use `asPath` here but instead `pathname`.

The difference is that asPath returns the external (public) route whereas the other returns the internal route. The idea of having `next-routes` is to decouple the routes from the code so if tomorrow we rename them, we just need to change the configuration in `routes.js`.

For this reason, I would prefer we use the internal routing as if we wouldn't know how is the route from the outside. You wouldn't need to make any other change.